pkg/settings/settings.go

```diff
@@ -244,6 +246,10 @@ func (settings *Settings) Verify() (err error) {
 		settings.Port = dbDefaultPorts[settings.DbType]
 	}
 
+	if settings.SSLMode == "" {
+		settings.SSLMode = "require"
```

Comment:
This will break backwards-compatibility for people using this tool out of the box - with this change it might suddenly stop working. Let's use `disable` as per default.